### DISCUSSION TITLE
Sine and Cosine at xml

### DISCUSSION THREAD

**Discussion by matheusdarocha** [JSBSim-Team/jsbsim#526] [Q&A] [answered]
Good night.

I want to write a script which the elevator deflection is a sine curve.
Does anyone know how to use sin and cosine at xml script?

thanks

**Comment by bcoconni** [comment, marked as answer]
I guess you can start from the script [c172_elevator_doublet.xml](https://github.com/JSBSim-Team/jsbsim/blob/master/scripts/c172_elevator_doublet.xml) which as far as I can tell does exactly what you are describing:
https://github.com/JSBSim-Team/jsbsim/blob/dd97e99032e6cce6ae3d0d91730f3c10f8ce94af/scripts/c172_elevator_doublet.xml#L94-L106
